tests/test_jointcal_lsstSim.py

```diff
@@ -54,32 +54,36 @@ def setUp(self):
     @unittest.skipIf(data_dir is None, "testdata_jointcal not setup")
     @unittest.skip('jointcal currently fails if only given one catalog!')
     def testJointCalTask_1_visits(self):
-        self._testJointCalTask(2, 0, absolute_error)
+        self._testJointCalTask(2, 0, dist_rms_absolute)
 
     @unittest.skipIf(data_dir is None, "testdata_jointcal not setup")
     def testJointCalTask_2_visits(self):
         # NOTE: The relative RMS limits were empirically determined from the
         # first run of jointcal on this data. We should always do better than
         # this in the future!
-        relative_error = 9.7e-3*u.arcsecond
-        self._testJointCalTask(2, relative_error, absolute_error)
+        dist_rms_relative = 9.7e-3*u.arcsecond
+        pa1 = 2.64e-3
+        self._testJointCalTask(2, dist_rms_relative, dist_rms_absolute, pa1)
```

Comment:
In general, I agree, but I think the intention of these magic numbers is pretty clear from the context of the method name.